lib/ClusterShell/NodeSet.py

```diff
@@ -916,6 +916,8 @@ def parse_group_string(self, nodegroup, namespace=None):
             namespace, group = grpstr.split(':', 1)
         if group == '*': # @* or @source:* magic
             reslist = self.all_nodes(namespace)
+        elif group.startswith('@'): # @@source group name list
+            reslist = self.grouplist(grpstr[1:])
```

Comment:
ok, no source name is supported because `grpstr[1:]` will be expanded to `''` and we are testing `namespace` as a boolean, so both `None` (official value)  and `''` will evaluate to `False`.